tests/helpers/percy.ts

```diff
@@ -52,8 +66,11 @@ const supportedFilters = /(@w1\s+|@w2\s+|@w3\s+|@h1\s+|@h2\s+|@h3\s+)/g;
     ...
   });
 */
-export default async function takeSnapshot(qunitAssert, options = {}) {
-  checkInput(qunitAssert, options);
+export default async function takeSnapshot(
+  qunitAssert: unknown,
```

Comment:
Somehow, the `assert` object, which originally should be of type `Assert` (from `qunit`, possibly extended by `qunit-dom`), is transformed into an object whose type definition I couldn't find:

<img width="540" alt="" src="https://user-images.githubusercontent.com/16869656/171866896-e7fcb097-ca4f-492e-b017-7ba431de715f.png">

I thought it'd be best to specify the type as `unknown`, then cast it into something (called `QUnitAssert` here) before we pass it to `getName()`.